tcollector.py

```diff
@@ -682,6 +689,33 @@ def send_data(self):
         # the packets out of the kernel's queue
 
 
+    def send_data_to_signalfx(self):
+        try:
+            gauges = []
+            for line in self.sendq:
+                split_line = self.add_tags_to_line(line).split()
+                metric, timestamp, value = split_line[:3]
+                tags = dict(tag.split('=') for tag in split_line[4:])
+                gauges.append({
+                    'metric': metric,
+                    'value': self.num(value),
+                    'timestamp': int(timestamp)*1000,
```

Comment:
How should I handle this? Do you know if there's a way for me to tell if it's configured to use seconds or milliseconds?